### DISCUSSION TITLE
MAC.IPTV.PLAYER.by.My-1 v4

### DISCUSSION THREAD

**Discussion by Cyogenus** [Cyogenus/IPTV-MAC-STALKER-PLAYER#5] [General]
Latest Update should work with most portals..... 

<hr /><em>This discussion was created from the release <a href='https://github.com/Cyogenus/IPTV-MAC-STALKER-PLAYER/releases/tag/'>MAC.IPTV.PLAYER.by.My-1 v4</a>.</em>

**Comment by Cyogenus** [comment]
I also created a Profile Manager to save your Portal url and logins but I haven't uploaded it to the repo yet been lazy 😆😆😆 If I get requests for this feature I'll update the code.

**Comment by MDN-DA** [comment]
https://www.virustotal.com/gui/file/378472a04d7f11fe794eb93c83cbc2438c548bea0d2688d719f5483c34357239

Hello,

I recently analyzed your program on VirusTotal, and I noticed that a few antivirus engines (e.g., Avast, AVG, McAfee) flagged it as potentially malicious (e.g., Win64:Evo-gen [Trj]). However, the majority of other engines marked it as clean or undetected.

Could you provide any clarification about why some antivirus programs might flag your software this way? Is it possible that certain heuristics or code patterns are triggering these alerts?

Thank you for your time and any insights you can offer!

Best regards,

**Reply by Cyogenus** [reply]
It's a Python script that has been converted into an executable (EXE) file. Some antivirus programs may flag the Python code as a virus because many viruses are created using Python. Even if you take the Python code and convert it to a Windows EXE yourself, it may still be detected as a virus. You're welcome to compile the Python script on your own if you'd like.
